source/css/patternfly.css

```diff
@@ -6104,9 +6110,66 @@ tbody.collapse.in {
     background: #fafafa;
     border-top-color: #d1d1d1; }
 
+.card-title {
+  overflow: hidden;
+  text-overflow: ellipsis; }
+
+.pf-card__block,
+.pf-card__header {
+  padding: 1.5rem; }
+
+.pf-card__footer {
+  padding: 1rem 1.5rem; }
+
+.pf-card__footer p {
+  margin: 0.25rem 0; }
+
+.pf-card__header__title {
+  margin: 0; }
+
+.pf-card__header__dropdown {
+  margin: -0.5rem 0; }
+
+.pf-card__kebab {
+  margin: -0.5rem; }
+
+.pf-card__top-element {
+  margin-bottom: 1.25rem; }
+
 .pf-card--accented {
   border-top: 2px solid #39a5dc; }
 
+.pf-card--select {
+  border: 2px solid transparent;
+  background-clip: content-box; }
+  .pf-card--select:hover::after {
+    box-shadow: 0 0.1em 0.3em rgba(3, 3, 3, 0.3); }
+
+.pf-card--select--active {
+  border-color: #39a5dc; }
+
+.pf-card--multiselect__block,
+.pf-card--multiselect__header {
+  padding-left: 2rem;
+  padding-right: 2rem; }
+
+.pf-card__checkbox {
+  width: 2rem;
+  height: 3rem;
```

Comment:
I changed the div in pf3 to a label for this case, and increased the width and height of the label to increase the clickable area associated with the checkbox